server.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.get("/ping", (req, res)=>{
+    res.send({"message":"pong"})
+});
+
+app.listen(port, ()=>{
```

Comment:
Consider adding error handling middleware to handle any errors that might occur in the application. This will improve the robustness of your server. [medium]